### PR TITLE
Ft cycles support 157110855

### PR DESCRIPTION
**Pull request by meshnesh**
# What does this PR do?

This PR allows for the allocation of different workout schedules to a work program.

## How should this be manually tested?

* Login as an `admin` with password `admin`
* Navigate to the Workout tab and create a work out program.
* Navigate to the Workout Schedule tab, select create a workout.
* Select the workout program you want to add the schedule to then save to populte the schedule to the page.

## What are the relevant pivotal tracker stories?

[#157110855](https://www.pivotaltracker.com/story/show/157110855)

### Screenshots(If applicable)

* Click on add work out

screen shot 2018-05-11 at 01 31 28

screen shot 2018-05-11 at 01 31 40

* Choose the workout 

screen shot 2018-05-11 at 01 31 48

* Choose the desired Schedule plan and save it

screen shot 2018-05-11 at 01 32 01

### CODE REVIEW

File: wger/manager/templates/schedule/view.html [hunk 1]
```diff
@@ -61,6 +61,46 @@
     $(function() {
         init_sortable();
     });
+
+    {% comment %} Function responsible for the displaying of the select form for cycles and number of weeks {% endcomment %}
```
Comment (betsybeth) [inline wger/manager/templates/schedule/view.html:65]: Good work on commenting.

File: wger/manager/views/schedule_step.py [hunk 2]
```diff
@@ -57,13 +58,28 @@ def get_form_class(self):
         This is defined here because only at this point during the request
         have we access to the current user
         '''
-
-        class StepForm(ModelForm):
-            workout = ModelChoiceField(queryset=Workout.objects.filter
-                                       (user=self.request.user))
+        class StepForm(ModelForm, forms.Form):
+            weeks = tuple(
```
Comment (betsybeth) [inline wger/manager/views/schedule_step.py:62]: I think you should add a docstring here to explain what the model does

**Review by betsybeth — DISMISSED** [review]

File: wger/manager/templates/schedule/view.html [hunk 3]
```diff
@@ -61,6 +61,46 @@
...
+            default:
+                $("#id_duration option").remove();
+                var count=1;
+                while( count <= 52){
+                    $("#id_duration").append("<option value="+count+">"+count+"weeks </option>");
+                    count++;
+                }
+                break;
+        }
+    };
```
Comment (FelixWambiri) [inline wger/manager/templates/schedule/view.html:102]: I like how you have comments that make it easier to follow through

**Review by FelixWambiri — APPROVED** [review]
Nice implementation and good work on the comments but you can add more comments especially in the schedule_step.py file

**Review by lakivisi-zz — APPROVED** [review]

**Pull request merged**